test/e2e/storage/volume_provisioning.go

```diff
@@ -1,784 +0,0 @@
-
-	v1 "k8s.io/api/core/v1"
-	rbacv1 "k8s.io/api/rbac/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/apiserver/pkg/authentication/serviceaccount"
-	clientset "k8s.io/client-go/kubernetes"
-	storageutil "k8s.io/kubernetes/pkg/apis/storage/util"
-	"k8s.io/kubernetes/test/e2e/feature"
-	"k8s.io/kubernetes/test/e2e/framework"
-	e2eauth "k8s.io/kubernetes/test/e2e/framework/auth"
-	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
-	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
-	"k8s.io/kubernetes/test/e2e/framework/providers/gce"
-	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
-	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-	"k8s.io/kubernetes/test/e2e/storage/testsuites"
-	"k8s.io/kubernetes/test/e2e/storage/utils"
-	admissionapi "k8s.io/pod-security-admission/api"
-)
-
-const (
-	// Plugin name of the external provisioner
-	externalPluginName = "example.com/nfs"
-)
-
-func checkGCEPD(volume *v1.PersistentVolume, volumeType string) error {
-	cloud, err := gce.GetGCECloud()
-	if err != nil {
-		return err
-	}
-	diskName := volume.Spec.GCEPersistentDisk.PDName
-	disk, err := cloud.GetDiskByNameUnknownZone(diskName)
-	if err != nil {
-		return err
-	}
-
-	if !strings.HasSuffix(disk.Type, volumeType) {
-		return fmt.Errorf("unexpected disk type %q, expected suffix %q", disk.Type, volumeType)
-	}
-	return nil
-}
-
-var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
-	f := framework.NewDefaultFramework("volume-provisioning")
-	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
-
-	// filled in BeforeEach
-	var c clientset.Interface
-	var timeouts *framework.TimeoutContext
-	var ns string
-
-	ginkgo.BeforeEach(func() {
-		c = f.ClientSet
-		ns = f.Namespace.Name
-		timeouts = f.Timeouts
-	})
-
-	f.Describe("DynamicProvisioner", framework.WithSlow(), feature.StorageProvider, func() {
-		ginkgo.It("should provision storage with different parameters", func(ctx context.Context) {
-
-			// This test checks that dynamic provisioning can provision a volume
-			// that can be used to persist data among pods.
-			tests := []testsuites.StorageClassTest{
-				// GCE/GKE
-				{
-					Name:           "SSD PD on GCE/GKE",
-					CloudProviders: []string{"gce", "gke"},
-					Timeouts:       f.Timeouts,
-					Provisioner:    "kubernetes.io/gce-pd",
-					Parameters: map[string]string{
-						"type": "pd-ssd",
-						"zone": getRandomClusterZone(ctx, c),
-					},
-					ClaimSize:    "1.5Gi",
-					ExpectedSize: "2Gi",
-					PvCheck: func(ctx context.Context, claim *v1.PersistentVolumeClaim) {
-						volume := testsuites.PVWriteReadSingleNodeCheck(ctx, c, f.Timeouts, claim, e2epod.NodeSelection{})
-						gomega.Expect(volume).NotTo(gomega.BeNil(), "get bound PV")
-
-						err := checkGCEPD(volume, "pd-ssd")
```

Comment:
Please do not remove this file. Remove all cloud provider calls instead (like `checkGCEPD()` here), but keep the test - we want to make sure that Kubernetes can still provision using in-tree GCE (AzureDisk, AWS EBS, ... storage classes) using CSI migration.
The test can't check that the right volume was created (`pd-ssd` in this case), still, it can check that at least something was provisioned and it's usable in a Pod.